components/LandingPage/LandingPage.tsx

```diff
@@ -4,14 +4,17 @@ import ImageMapper from 'react-img-mapper'
 import map from '@/lib/image-map.json'
 import { useMeasure } from 'react-use'
 import { useState } from 'react'
+import Image from 'next/image'
 
 const LandingPage = () => {
   const [containerRef, { height }] = useMeasure() as any
   const [isVisibleToolTip, setIsVisibleTooltip] = useState(false)
   const [tooltipX, setTooltipX] = useState(0)
   const [tooltipY, setTooltipY] = useState(0)
+  const [isVisibleDlg, setIsVisibleDlg] = useState(false)
```

Comment:
try not to use `shorthand` abbreviations 
- why? harder for new devs to understand what variables do
```suggestion
  const [isDialogOpen, setIsDialogOpen] = useState(false)
```